elasticdl/python/common/model_handler.py

```diff
@@ -157,3 +171,36 @@ def _replace_embedding_attributes_for_subclass(
                 )
                 setattr(model, name, embedding_layer)
         return model
+
+    def _get_trained_params(self, model):
+        """Get all trained variable values of the model
+        """
+        trained_params = self._get_non_embedding_variables(
+            -1, elasticdl_pb2.MINIMUM
+        )
+        trained_embedding_params = self._get_trained_embedding_params(model)
+        trained_params.update(trained_embedding_params)
+        return trained_params
+
+    def _get_trained_embedding_params(self, model):
+        """Get trained embedding table from PS
+        """
+        embedding_params = {}
+        embedding_layers = model_utils.find_layer(model, Embedding)
+        for embedding_layer in embedding_layers:
+            # TODO get all embedding vectors of the embedding layer from PS
+            pass
+        return embedding_params
+
+    def _get_non_embedding_variables(self, version, method):
```

Comment:
We can add TODO here.
If the new PS is online, we need get the non embedding variables from PS instead of master.